### ISSUE TITLE
Shopify serve / ngrok error

### ISSUE THREAD

**Issue by HauntedSmores** [Shopify/shopify-app-cli#1047]
# Issue summary

Write a short description of the issue here ↓

Repeated error message when running `shopify serve` and navigating to ngrok url.

## Expected behavior

No error when navigating to ngrok local app.

## Actual behavior

When navigating to ngrok address this error is repeatedly emitted in the console.

```
Error: aborted
┃       at connResetException (node:internal/errors:631:14)
┃       at abortIncoming (node:_http_server:578:17)
┃       at socketOnEnd (node:_http_server:593:5)
┃       at Socket.emit (node:events:388:22)
┃       at endReadableNT (node:internal/streams/readable:1305:12)
┃       at processTicksAndRejections (node:internal/process/task_queues:80:21)
```

## Steps to reproduce the problem

1. `shopify create node
2. `shopify serve`
3. Install custom app and navigate to given ngrok url

## Specifications

- App type: custom app
- Operating System: mac os Big Sur 11.1
- Shell: zsh
- Ruby version (ruby -v): ruby 2.6.3p62
- Node v15.5.1
- npm v7.3

**Comment by tombohub** [comment]
yes, it says to run the shopify serve when it's obviously running

**Comment by ralphhermeling** [comment]
I'm having the same problem

**Comment by fotoflo** [comment]
me too

**Comment by ScottJr** [comment]
Same here, multiple clean installs on OSX.

**Comment by jcneshi** [comment]
Same

**Comment by ScottJr** [comment]
@thecodepixi - It for sure still exists. It isn't stale, it's unsolved.
